src/Core/Entity/Message.php

```diff
@@ -61,7 +61,7 @@ protected function getDefinition()
 			'user_id'    => 'int',
 			'title'      => 'string',
 			'message'    => 'string',
-			'datetime'   => '*timestamp',
+			'datetime'   => '*date',
```

Comment:
I haven't completely verified this but pretty sure it should work. It matches the date handling on `Post` for the `post_date` field